dockerfiles/jupyter/README.ipynb

```diff
@@ -8,29 +8,71 @@
     "\n",
-    "Check [**MLRun Documentation**](https://docs.mlrun.org/en/latest/index.html) for further details.\n",
+    "MLRun supports numerous real-time and batch runtimes, described in [Kinds of functions (runtimes)](https://docs.mlrun.org/en/stable/concepts/functions-overview.html#function-runtimes). \n",
+    "The different function runtimes takes care of automatically transforming the code and spec to fully managed and elastic services over Kubernetes, which saves significant operational overhead, addresses scalability, and reduces infrastructure costs. The function parameters and capabilities are explained in more detail in [Create and use functions](https://docs.mlrun.org/en/stable/runtimes/create-and-use-functions.html#create-and-use-functions).\n",
     "\n",
-    "## Configure Your Environment\n",
+    "MLRun enables users to manage and log artifacts, which are data objects produced and/or consumed by functions, jobs, or pipelines.\n",
+    "Each artifact includes metadata attributes such as tags, data previews, and model metrics, providing essential insights for managing machine learning (ML) and large language model (LLM) projects. These attributes help users track, review, and organize different artifacts efficiently.\n",
     "\n",
-    "MLRun  backend service can run locally or over Kubernetes (preferred), see the instructions for installing it [**locally** using Docker](https://docs.mlrun.org/en/latest/install/local-docker.html) or [over **Kubernetes** Cluster](https://docs.mlrun.org/en/latest/install/kubernetes.html). Alternatively, you can use [Iguazio's **managed** MLRun service](https://www.iguazio.com/docs/latest-release/).\n",
+    "**For further details please check -**\n",
+    " * MLRun Documentation - [link](https://docs.mlrun.org/en/stable/index.html)\n",
+    "    * Projects Documentation - [link](https://docs.mlrun.org/en/stable/projects/project.html)\n",
+    "    * Functions Documentation - [link](https://docs.mlrun.org/en/stable/runtimes/functions.html)\n",
+    "    * Artifacts Documentation - [link](https://docs.mlrun.org/en/stable/store/artifacts.html)\n",
+    " * Quick start tutorial for machine learning tutorial - [link](https://docs.mlrun.org/en/stable/tutorials/01-mlrun-basics.html)\n",
+    " * MLRun cheat sheet - [link](https://docs.mlrun.org/en/stable/cheat-sheet.html#mlrun-setup)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ddeb6a5c",
+   "metadata": {},
+   "source": [
+    "### MLRun CE default components\n",
     "\n",
-    "This Jupyter Notebook server is designed to work with the different options, it works out of the box with the **local** deployment mode.<br> \n",
-    "In order to work with remote MLRun service (Kubernetes or managed) you need to edit and save the [**mlrun.env**](./mlrun.env) file in the following way:\n",
+    "The MLRun CE (Community Edition) chart includes the following components below by default:\n",
```

Comment:
"The MLRun CE (Community Edition) chart, by default, includes the following components:\n",